### DISCUSSION TITLE
Hospital stay [days] = 3 (is this for ICU flow + counter flow?)

### DISCUSSION THREAD

**Discussion by tiagoandresvaz** [neherlab/covid19_scenarios#703] [General] [answered]
About the average number of days, a severe case stays in regular hospital beds (epidemiology parameter), which is preset by default for the USA integrated scenario with 3 days.

In data/scripts/model.py it is considering duration for both periods (before and after) the patient goes to ICU.  

dpop[Sub.H]  = +flux_I_H + flux_C_H - flux_H_R - flux_H_C
dpop[Sub.C]  = +flux_H_C - flux_C_D - flux_C_H

Why you are setting default as 3 days, isn't it too low according to other USA references? 
Example: 4.1 days (3.8 days for discharged and 5.1 days for deaths)
https://jamanetwork.com/journals/jama/fullarticle/2765184/

**Comment by rneher** [comment, marked as answer]
The policies on when a patient gets admitted to a hospital (as opposed to outpatient care) or when/if a patient is moved to ICU differ from place to place. These numbers are meant as a starting point for adjustment by the user. But we can update them to different presets (e.g. 4 days instead of 3).

And you are right that the patient flux in our model is `regular ward` to `ICU` and back to `regular ward`. The complexities of different disease courses are not covered by simple models like ours. But if you have specific suggestions, let us know.

**Comment by tiagoandresvaz** [comment]
Prof @rneher thanks for open this space. 

In southern Brazil, we have measured around 17% of cases (>70 years) are going from regular ward direct to death. We believe that it is due to palliative care on very severe cases with death in less than 24 hrs and possibly other characteristics from our public healthcare system.  

I am bringing as a suggestion the possibility of an adaptation to include this new flow (green arrow) shown in the attached image. Adding a new column to the interface to set "Fatal % of Severe" age-group-specific parameters. Adjusting the model to:

        flux_H_D = params.rates.critical*params.fracs.fatality_hospital*pop2d[Sub.H]

and

        dpop[Sub.S]  = -flux_S
        dpop[Sub.E1] = +flux_S  - flux_E1
        dpop[Sub.E2] = +flux_E1 - flux_E2
        dpop[Sub.E3] = +flux_E2 - flux_E3
        dpop[Sub.I]  = +flux_E3 - flux_I_R - flux_I_H
        dpop[Sub.H]  = +flux_I_H + flux_C_H - flux_H_R - flux_H_C - flux_H_D
        dpop[Sub.C]  = +flux_H_C - flux_C_D - flux_C_H
        dpop[Sub.R]  = +flux_H_R + flux_I_R
        dpop[Sub.D]  = +flux_C_D + flux_H_D

Our team is working to load data from the [State of Rio Grande do Sul ](https://distanciamentocontrolado.rs.gov.br/) and we have implemented a parser and other customs with successes using our fork. Do you think that this new suggested flow is an effort that may be of interest to all?

![SIR br](https://user-images.githubusercontent.com/752397/85744378-3c100f80-b6db-11ea-9a9b-2a2b915f75db.png)

**Reply by rneher** [reply]
Hi @tiagoandresvaz,

I think this is a good suggestion. we have been hesitant to introduce too many additional parameters as it makes model fitting difficult and doesn't necessarily increase interpretability. But death straight from a regular ward (or nursing home) makes sense. This could in principle be an age dependent parameter I suppose. 

Did you want to make a PR with these suggestions?

best,
richard

**Comment by tiagoandresvaz** [comment]
Prof. @rneher PR done!

https://github.com/neherlab/covid19_scenarios/pull/751

Please let me know if I can help to set the documentation accordingly! 

Thanks,
Tiago
